components/SearchResult.vue

```diff
@@ -38,11 +38,11 @@ defineProps({
     spokenLanguages: {
         type: Array,
         required: true,
-        validator(arrayValue) {
+        validator(arrayValue: Array<any>) {
```

Comment:
It would be better to use generics but I am guessing this section may get refactored after the backend is connected so ain't bothering

Reply:
`any` is fine for now 👍